src/item_contents.cpp

```diff
@@ -607,7 +608,9 @@ units::volume item_contents::max_containable_volume() const
 {
     units::volume ret = 0_ml;
     for( const item_pocket &pocket : contents ) {
-        if( !pocket.is_type( item_pocket::pocket_type::CONTAINER ) || pocket.is_ablative() ) {
+        // pockets that aren't traditional containers or don't have a default value shouldn't be counted for this
+        if( !pocket.is_type( item_pocket::pocket_type::CONTAINER ) || pocket.is_ablative() ||
+            pocket.holster_full() || pocket.volume_capacity() > 199999999_ml ) {
```

Comment:
You can do this without magic numbers
```suggestion
            pocket.holster_full() || pocket.volume_capacity() >= pocket_data::max_volume_capacity ) {
```
and define
```
static constexpr units::volume max_volume_capacity = 200000000_ml;
units::volume volume_capacity = max_volume_capacity;
```
in `item_pocket.h:377`
and
`constexpr units::volume pocket_data::max_volume_capacity;`
in `item_pocket.cpp:57`.
Same for the other volume and mass checks below.

Reply:
Honestly it was getting late and I just didn't want to edit headers and compile again. I'll clean up the magic numbers tomorrow morning when I have more time.